src/panels/lovelace/views/hui-masonry-view.ts

```diff
@@ -98,11 +98,13 @@ export class MasonryView extends LitElement implements LovelaceViewElement {
   }
 
   protected firstUpdated(): void {
+    this._updateColumns = this._updateColumns.bind(this);
```

Comment:
I dont understand this part

Reply:
It would not use the context of the class, as the event listener would call it, it is the same as `() => this._updateColumns()`

Reply:
Oh that makes sense. 👍